### ISSUE TITLE
Unneccessary margin for images?

### ISSUE THREAD

**Issue by Dyvd4** [rpearce/react-medium-image-zoom#352]
## Issue Type

- [x] question / bug

## Description

I want to ask if this margin is neccessary. When using the zoom component with an image this adds (in this case) unneccessary margin.
In the example I just have some images aligned with flexbox (column).

![margin](https://user-images.githubusercontent.com/72784312/187714542-9dee6044-4e14-48aa-80c6-78236febaca4.jpg)
![maring_css](https://user-images.githubusercontent.com/72784312/187714548-6d8e749f-5ac8-4161-9a33-14cbd9d0cc23.jpg)
![wo_margin](https://user-images.githubusercontent.com/72784312/187714734-f3eacfe9-b3ad-4177-bbd3-26ae8077e5a1.jpg)

**Comment by rpearce** [comment]
@Dyvd4 I can't read what any of those images say, as the images are too small.

If there's an issue that you can provide steps to help me reproduce, then I can be of more help!

**Comment by Dyvd4** [comment]
These images are a bit misleading. They contain errors related to my project, not to the image zoom. There could be also any other image. The real "problem" here is the left margin.  You can see the difference in the screenshots. In the second screenshot there is the left margin disabled. The margin comes from some of your css classes.

**Comment by rpearce** [comment]
@Dyvd4 Honestly, I'm having a hard time understanding. Let's back up:

* What version of `react-medium-image-zoom` are you using?
* 2 out of the 3 screenshots (the error ones) above are illegible, and if they aren't related to this project, could you help me understand why they're here?
* The other image doesn't appear to have any CSS selectors that `react-medium-image-zoom` uses. Is it possible your other tools are doing this? I've never used `margin-inline-end` nor `margin-inline-start` on any project in my life, so those aren't coming from this tool.

Feel free to reply, but as this doesn't seem to be an issue with this project, I'm going to close the issue.

**Comment by Dyvd4** [comment]
@rpearce As I said before. These images are dummy images. Coincidentally they are errors. But these errors aren't related to this issue, so you don't have to care about what they say.

Furthermore I am using version 5.0.2.

And last but not least I just wanted to ask if the margin comes from your library. In my project I am using ckakra ui with tailwindcss as ui libraries. I have a list of image attachments which I render (what you can see in the screenshots). When I wrapped them in the Zoom component, this left margin I showed in the second screenshot was added so I thought it comes from your library because if I remove the Zoom component, the margin is gone. 

To sum it up - if you say you don't use this margin in your library then it is okay, that's all I wanted to know. That's also because my issue type is question / bug because I wasn't sure. Maybe chakra ui does apply some styles on it (idk why it should because it doesn't come with any preflight).
Anyways, thanks for the replies, your library is great (no toxic irony)!

**Comment by rpearce** [comment]
@Dyvd4 Thank you for clarifying! I don't believe it is coming from here, and thanks for the kind words.
